Makefile

```diff
@@ -157,7 +157,7 @@ gen-k8s-check: $(apis_sources)
 	touch $@
 
 components:
```

Comment:
The number of components is low is not like we add new componentes everyweek so we should have something like 
`make kubemacpool-bump` and friends and also `make all-components-bump`

Reply:
I was thinking of changing bump-all.sh to accept COMPONENTS arg so it would itterate on them